data/pages-content/thankyou.md

```diff
@@ -1,370 +1,375 @@
 
 ## Code of Conduct Committee
 
-Here are the humans that receive & investigate Code of Conduct reports and help
+These humans are responsible for receiving & handling Code of Conduct reports and help
 the EuroPython community remain a safe and friendly place:
 
-- Silvia Uberti
 - Naomi Ceder
+- Vicky Twomey-Lee
+- Cheuk Ting Ho
+- Sebastiaan Zeeff
+- Silvia Uberti
 - Anders Hammarquist
-- Kevin O'Brien
 
-## Communications & Design
+## Communications
 
-When you read the (hopefully clear but definitely witty) content on our website,
-blog, newsletter & social media, it was written by one of these wordsmith from
-our Communications team:
+As you peruse the (hopefully lucid but always witty) content on our website, blog, newsletter, and social media, it was created by one of the wordsmiths/meme crafters from our Communications team:
 
+- Sangarshanan
 - Vaibhav Srivastav (VB)
 - Diego Russo
-- Sangarshanan
-- Artur Czepiel
-- Laís Carvalho
-- Vicky Twomey-Lee
-- Yasmin Morshed
+- Nicholas Tollervey
 
-And the small and big design stuff you saw including t-shirts, badges, banners,
-stickers and social cards were scrambled together with love by our Design team:
 
-- Laís Carvalho
+## Design
+
+### Conference Designers
+
+Our logo, website assets, t-shirts, banners, badges, lanyards, and general aesthetics of EuroPython 2023 were created by our designers
+- [Anastasia Santoro](https://www.linkedin.com/in/anastasia-santoro-ui-ux-designer-620316157/)
+- [Orlando Festa](https://www.linkedin.com/in/orlandofesta/)
+<!-- - FIXME - ask Patrick if we should put them down below in the Acknowledgement.  -->
+
+### Design Team
+And the rest of the assets, stickers and social cards were scrambled together with love by our Design team.
 - Patrick Arminio (our front-end wizard)
 - Raquel Dou
 
 ## Financial Aid team
 
-If you are one of the 94 people who participated EuroPython 2022 either in
-Dublin or remotely, you have surely exchanged many emails with some of our
-Finaid team members. From spreadsheet wizardry, to responding to countless
-emails and processing all the receipts, here are our kind and organised humans
-behind the Finaid team:
+If you are one of the 107 people who participated in EuroPython 2023 in Prague or remotely, you have surely exchanged many emails with some of our Financial Aid team members. From spreadsheet wizardry, to responding to countless
+emails and processing all the receipts, here are our kind and organised humans behind the Finaid team:
 
 - Sebastiaan Zeff
-- Cheuk Ting Ho
 - Vicky Twomey-Lee
-- Johanna B. Lunghini
-
-## Infrastructure Team
-
-From mailing to conferencing tooling and website, we undertook a big
-infrastructure overhaul this year thanks to our amazing Infrastructure team.
-These humans quietly deploy and manage our services, and are perhaps often
-overlooked until things break:
-
-- Artur Czepiel
-- Silvia Uberti
-- Patrick Arminio
-- Francesco Pierfederici
+- Cheuk Ting Ho
+- Mannie Young
+- Manoj Pandey
 
```

Comment:
Add Raquel Dou here.

Reply:
A bit confused. Where should I add Raquel to?